pulsar-client-kafka-compat/pulsar-client-kafka/pom.xml

```diff
@@ -62,6 +62,14 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
```

Comment:
what are the changes for?

Reply:
These imports were missing for the pulsar-client-kafka module.  A test in this module was failing because of a classloader powermock issue but  I could not compile this module by itself without these dependencies.  Somehow there is no issue if you compile the whole pulsar project.

Reply:
if so, can you do it in a separated pull request?

Reply:
sure